src/actions/AppActions.ts

```diff
@@ -328,7 +328,7 @@ async function createAccountForContract(contractName: string) {
   const keyPair = await app.state.keyStore.getKey(contractName);
   // if no keypair in keystore, it means the account does not exist.
   // maybe there is a better way to check this?
-  if (!keyPair.getPublicKey()) {
+  if (!keyPair || !keyPair.getPublicKey()) {
```

Comment:
is `!keyPair.getPublicKey()` still needed?

Reply:
I think it can't happen, so it's there only for pedantic reasons. I am fine removing it.

Reply:
Actually, I am not sure which version of nearlib master is so -- removing it without also bumping nearlib will actually break functionality. Let's do it with the nearlib version update PR